timelock/src/lib.rs

```diff
@@ -0,0 +1,210 @@
+// This contract demonstrates 'timelock' concept and implements a
+// greatly simplified Claimable Balance (similar to
+// https://developers.stellar.org/docs/glossary/claimable-balance).
+// The contract allows to deposit some amount of token and allow another
+// account(s) claim it before or after provided time point.
```

Comment:
:bulb: The `//!` prefix on a comment acts as a comment for the module / crate, and shows up in docs. We don't generate docs for these, but feels appropriate to use that style of comment for the comment I see here, which seems to describe the crate as a whole.
```suggestion
//! This contract demonstrates 'timelock' concept and implements a
//! greatly simplified Claimable Balance (similar to
//! https://developers.stellar.org/docs/glossary/claimable-balance).
//! The contract allows to deposit some amount of token and allow another
//! account(s) claim it before or after provided time point.
```

Reply:
Done.